lib/cfa/base_model.rb

```diff
@@ -53,14 +53,15 @@ def load
     # smart to at first modify existing value, then replace commented out code
     # and if even that doesn't work, then append it at the end
     # @note prefer to use specialized methods of children
-    def generic_set(key, value)
-      modify(key, value) || uncomment(key, value) || add_new(key, value)
+    def generic_set(key, value, tree = data)
```

Comment:
this API change is needed to allow generic set on subsection. It is needed to fix uncommenting commented value. Problem with current one is that key is only top-level, but do not allow to set nested value.